usr/lib/systemd/system/apisix.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=apisix
+Conflicts=apisix.service
+After=network.target
```

Comment:
See https://www.freedesktop.org/wiki/Software/systemd/NetworkTarget/
Should be network-online.target

Reply:
Thank you! Fixed